stdlib/public/core/Integers.swift.gyb

```diff
@@ -1171,6 +1192,10 @@ ${operatorComment(x.operator, False)}
 /// declaring conformance to the protocol and ensuring that the values of your
 /// type support negation. To customize your type's implementation, provide
 /// your own mutating `negate()` method.
+///
+/// When the additive inverse of a value is unrepresentable in a conforming
+/// type, the operation should return a sentinel value, such as NaN, or
```

Comment:
A type conforming to FloatingPoint is always symmetric about zero, so NaN should never result like this.